components/Modal/CreateImageUpload.js

```diff
@@ -374,11 +429,75 @@ class CreateImageUploadModal extends React.Component {
       imageName,
       imageType,
       imageSize,
+      ostreeSettings,
       minImageSize,
       maxImageSize,
       uploadService
     } = this.state;
 
+    const ostreeFields = (
+      <React.Fragment>
+        <div className="pf-c-form__group">
+          <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+            <label htmlFor="ostree-parent-input" className="pf-l-flex__item">
+              <span className="pf-c-form__label-text">
+                <FormattedMessage defaultMessage="Parent commit" />
+              </span>
+            </label>
+            <Popover
+              id="ostree-parent-popover"
+              bodyContent={
+                <FormattedMessage
+                  defaultMessage="Provide the ID of the latest commit in the updates repository for which this commit provides an update."
+                />
+              }
+              aria-label={formatMessage(ariaLabels.ostreeParent)}
+            >
+              <Button variant="plain" aria-label={formatMessage(ariaLabels.ostreeParent)}>
+                <OutlinedQuestionCircleIcon id="popover-icon" />
+              </Button>
+            </Popover>
+          </div>
+          <TextInput
+            className="pf-c-form-control"
+            value={ostreeSettings.parent !== undefined ? ostreeSettings.parent : ""}
+            type="text"
+            id="ostree-parent-input"
+            onChange={this.setOstreeParent}
+          />
+        </div>
+        <div className="pf-c-form__group">
+          <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+            <label htmlFor="ostree-ref-input" className="pf-l-flex__item">
+              <span className="pf-c-form__label-text">
+                <FormattedMessage defaultMessage="Branch" />
```

Comment:
Did you mean to change this to "Ref"? I thought when chatting with @gicmo we discussed displaying "Ref" as the input label, and explaining that it is a branch in the help text.

Reply:
Thanks for the catch. I updated this to `Ref` now.